tools/deployment/package-helm/values.yaml

```
@@ -60,8 +60,7 @@ image:
 #     the same path).
 distributedDeployment: false
 
-# Number of concurrent processes per worker pod.
-workerConcurrency: 8
+
 
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)